integrations/facebook-pixel/test/index.test.js

```diff
@@ -1942,6 +2005,58 @@ describe('Facebook Pixel', function() {
         );
         assertEventId(window.fbq);
       });
+
+      it('should send facebook pixel properties for product purchase', function() {
+        analytics.track(
+          'Order Completed',
+          {
+            products: [
+              {
+                product_id: '507f1f77bcf86cd799439011',
+                name: 'Monopoly: 3rd Edition',
+                price: 19,
+                quantity: 1
+              },
+              {
+                product_id: '505bd76785ebb509fc183733',
+                name: 'Uno Card Game',
+                price: 3,
+                quantity: 2
+              }
+            ],
+            currency: 'USD',
+            total: 0.5,
+            name: 'my product'
+          },
+          {
+            'Facebook Pixel': {
+              contentType: 'myCustomType'
+            }
+          }
+        );
+        analytics.called(
+          window.fbq,
+          'trackSingle',
+          options.pixelId,
+          'Purchase',
+          {
+            content_ids: [
+              '507f1f77bcf86cd799439011',
+              '505bd76785ebb509fc183733'
+            ],
+            content_type: ['myCustomType'],
+            currency: 'USD',
+            value: '0.50',
+            content_name: 'my product',
```

Comment:
I don't think we need to map `content_name` here.

Reply:
Got that, **content_name** is missing field but we only need to map it if its corresponding field present user specs.
Also changed **total** to **revenue** in analytics mapping